docs/realm.js

```diff
@@ -119,6 +119,16 @@ class Realm {
      */
     static openAsync(config, callback, progressCallback) {}
 
+    /**
+     * Open the default Realm. The server URL for the current user will be used as base for
+     * the URL for the synced Realm. If {Realm.Sync} is not enabled, the default local Realm will
+     * be opened.
+     * @throws {Error} if zero or multiple users are logged in
+     * @returns {ProgressPromise} - a promise that will be resolved with the Realm instance when it's available.
+     * @since 2.3.0
+     */
+    static default() {}
```

Comment:
This naming and behavior seems very confusing. There's no indication that what you're getting back is a synced Realm, and it will sometimes give back a non-synced Realm?

This pattern of providing a factory method was discussed by other bindings on Slack, and we instead decided to provide a factory for a _configuration_, since that makes it much more flexible to use, and allows using it for other places that need a configuration.